sbin/unit-tests

```diff
@@ -45,8 +44,8 @@ help() {
 sanitizer_defs() {
 	if [[ -n $SAN ]]; then
 		ASAN_LOG=${LOGS_DIR}/${TEST_NAME}.asan.log
-		export ASAN_OPTIONS="detect_odr_violation=0:alloc_dealloc_mismatch=0:halt_on_error=0:detect_leaks=1:log_path=${ASAN_LOG}"
-		export LSAN_OPTIONS="suppressions=$ROOT/tests/memcheck/asan.supp"
+		export ASAN_OPTIONS="detect_odr_violation=0:alloc_dealloc_mismatch=0:halt_on_error=0:detect_leaks=1:log_path=${ASAN_LOG}:verbosity=1:log_thread=1"
+		export LSAN_OPTIONS="suppressions=$ROOT/tests/memcheck/asan.supp:verbosity=1:log_thread=1"
```

Comment:
did you find out what log_thread=1 is doing?

Reply:
No... But I assume it means they are internally using a thread for writing to the log (for performence reasons, for not blocking when doing i/o). Anyway, I added it because they said so, I guess it's a better practice.